lighthouse-core/test/lib/url-shim-test.js

```diff
@@ -335,4 +335,30 @@ describe('URL Shim', () => {
     assert.ok(!URL.isNonNetworkProtocol('http'));
     assert.ok(!URL.isNonNetworkProtocol('ws'));
   });
+
+  describe('guessMimeType', () => {
+    it('handles invalid url', () => {
+      expect(URL.guessMimeType('')).toEqual(undefined);
+      expect(URL.guessMimeType('I_AM_NO_URL')).toEqual(undefined);
+    });
+
+    it('uses mime type from data URI', () => {
+      expect(URL.guessMimeType('data:image/png;DATA')).toEqual('image/png');
+      expect(URL.guessMimeType('data:image/jpeg;DATA')).toEqual('image/jpeg');
+      expect(URL.guessMimeType('data:image/jpg;DATA')).toEqual('image/jpg');
```

Comment:
this is an interesting one, should we use *any* `image/` MIME type even if it's not valid? I'm fine narrowing scope and doing whatever you think is more straightforward @adamraine

Reply:
I'm thinking get rid of `image/jpg` support, I'd rather surface `undefined` for unsupported mime types.